src/routes/settings/(nav)/assistants/[assistantId]/+page.svelte

```diff
@@ -194,6 +199,26 @@
 		</div>
 	</div>
 
+	<div>
+		<h2 class="b-2 inline font-semibold">Embed this assistant</h2>
```

Comment:
just my opinion but I think this is kind of an advanced feature and it's very visible in the UI of assistants atm

Maybe a button that copies the code to clipboard with an explainer text would be enough and take less space ?

Reply:
As it can grow the usage of HuggingChat I think it's fine if it's stays well visible.

Reply:
yes same